AllReadyApp/Web-App/AllReady/Services/Routing/GoogleOptimizeRouteService.cs

```
@@ -32,7 +32,8 @@ public GoogleOptimizeRouteService(IOptions<MappingSettings> mappingSettings, ILo
                 try
```

@chinwobble let it throw an exception